packages/lib-bitcoinjs/index.ts

```diff
@@ -23,13 +25,32 @@ const assertSigned = (res: GenerateSignatureResponse) => {
   }
 }
 
+const xOnlyPubkey = (publicKey: Buffer) => {
+  return publicKey.subarray(1, 33)
+}
+
+const getTweakedPublicKey = (publicKey: Buffer, taprootMerkleRoot?: Buffer) => {
+  const xOnlyPublicKey = xOnlyPubkey(publicKey)
+  const tweakedKey = tweakKey(xOnlyPublicKey, taprootMerkleRoot)
+  return Buffer.concat([Uint8Array.from([tweakedKey!.parity + 2]), tweakedKey!.x])
+}
+
 export class DfnsWallet implements SignerAsync {
   private readonly dfnsClient: DfnsApiClient
-  public readonly publicKey: Buffer
+  public readonly scheme: GetWalletResponse['signingKey']['scheme']
+  public publicKey: Buffer
+  public internalPubkey: Buffer
```

Comment:
it's nullable since we don't set it if not schnorr.

```suggestion
  public internalPubkey?: Buffer
```